blockbase/sass/base/_utility.scss

```diff
@@ -3,7 +3,7 @@
 //Apply to Image block when we don't want margin bottom before caption
 .image-no-margin {
 	margin: 0;
-	& > * { //the first sibling is a div on the editor and an image on the frontend
+	& * { //the first sibling is a div on the editor, and an image or a link on the frontend
```

Comment:
I tried out a few different fixes for this but went with this as it had the least amount of changes. Some of the others were:

- Applying `font-size: 0;` to the children
- Applying `display: inherit;` or `display: block` to the children

Reply:
I think this makes sense, but I'd remove the comment or edit it since we are not targeting just the first sibling now

Reply:
Thanks Maggie, I've updated the comment so it doesn't mention the first sibling anymore.